README.md

```diff
@@ -1,44 +1,103 @@
-# <img src="../../../assets/icon.png" height="10%" width="10%" /> Echo Bot
+# Overview of the AI Chat Bot template
 
-Teams AI Conversation Bot sample for Teams.
+This template showcases a bot app that responds to user questions like an AI assistant. This enables your users to talk with the AI assistant in Teams to find information.
 
-This sample shows how to incorporate basic conversational flow into a Teams application. It also illustrates a few of the Teams specific calls you can make from your bot.
+The app template is built using the Teams AI library, which provides the capabilities to build AI-based Teams applications.
 
-- [Concepts](#concepts)
-- [Prerequisites](#prerequisites)
-- [Run](#run)
-- [Development](#development)
+- [Overview of the AI Chat Bot template](#overview-of-the-ai-chat-bot-template)
+  - [Get started with the AI Chat Bot template](#get-started-with-the-ai-chat-bot-template)
+    <!-- - [Use Azure OpenAI](#use-azure-openai) -->
+  - [What's included in the template](#whats-included-in-the-template)
+  - [Extend the AI Chat Bot template with more AI capabilities](#extend-the-ai-chat-bot-template-with-more-ai-capabilities)
+  - [Additional information and references](#additional-information-and-references)
 
-## Concepts
+## Get started with the AI Chat Bot template
 
-- Listening/Sending Activities
-- Error Handling
+> **Prerequisites**
+>
+> To run the AI Chat Bot template in your local dev machine, you will need:
+>
+> - [Node.js](https://nodejs.org/), supported versions: 16, 18
+> - [Python](https://www.python.org/), version 3.11 or higher
+> - [Python extension](https://code.visualstudio.com/docs/languages/python), version v2024.0.1 or higher
+> - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [Teams Toolkit CLI](https://aka.ms/teamsfx-cli)
+> - An account with [OpenAI](https://platform.openai.com/).
 
-![Screenshot](./assets/screenshot_0.png)
+1. First, select the Teams Toolkit icon on the left in the VS Code toolbar.
+1. In file *env/.env.testtool.user*, fill in your OpenAI key `SECRET_OPENAI_API_KEY=<your-key>` and `SECRET_OPENAI_MODEL_DEPLOYMENT_NAME=<your-model-name>`.
```

Comment:
The default launch config is testtool?

Reply:
will consider it

Reply:
Please review the latest Readme file to see if it is comfortable to you.